test/elpa.el

```diff
@@ -0,0 +1,5 @@
+(setq package-user-dir
+      (expand-file-name (format ".cask/%s/elpa" emacs-version)))
```

Comment:
I'm aware of the current issue with Emacs snapshot but that's an issue in Emacs (Emacs 25 rc works fine) and beyond our control. Unfortunately that happens from time to time 😒